docs/madr/decisions/034-default-target-ref-policy.md

```diff
@@ -0,0 +1,135 @@
+# TargetRef policies as a default
+
+* Status: accepted
+
+Technical Story: https://github.com/kumahq/kuma/issues/8467
+
+## Context and Problem Statement
+
+At present, Kuma has two policy types:
+
+* Legacy policies with Source and Destination attributes
+* New policies based on TargetRef
+ 
+The legacy policies currently serve as the defaults. However, starting from release 2.6, the intention is to switch the default policy to TargetRef.
+
+## Considered Options
+
+1. Prefer avoiding the creation of default policies and utilize plugin code in cases where there are no existing default legacy policy.
+2. Introduce a `policyEngine` field within the `Mesh` object, allowing the selection of the default policy version.
+
+## Decision Outcome
+
+The decision is to stop creating default policies and utilize plugin code in cases where there are no existing default legacy policy.
```

Comment:
can you go into the specifics of `utilize plugin code` ? do you mean creating envoy resources needed if we detect no default policy? is that easy to do?

Reply:
By `utilize plugin code` i mean pluggable policies (targerRef)

Reply:
Basically just hardcode XDS config regardless if a resource is present or not, right?

Reply:
Tried to clarify it a bit, if we don't have an old policy, code related to the targetRef policies generates configuration even if there is no default new policy, which makes everything work. This is the difference to old policies because when there is no default policy means there is no traffic.

Reply:
I'm still not entirely clear on this part. I think it would be helpful if you could specify that you're referring specifically to the code that manages traffic flow.